src/structures/ApplicationCommand.js

```diff
@@ -126,62 +140,7 @@ class ApplicationCommand extends Base {
    *   .catch(console.error);
    */
   delete() {
-    return this.manager.delete(this);
-  }
-
-  /**
-   * Data for setting the permissions of an application command.
-   * @typedef {Object} ApplicationCommandPermissionData
-   * @property {Snowflake} id The ID of the role or user
-   * @property {ApplicationCommandPermissionType|number} type Whether this permission is for a role or a user
-   * @property {boolean} permission Whether the role or user has the permission to use this command
-   */
-
-  /**
-   * The object returned when fetching permissions for an application command.
-   * @typedef {Object} ApplicationCommandPermissions
-   * @property {Snowflake} id The ID of the role or user
-   * @property {ApplicationCommandPermissionType} type Whether this permission is for a role or a user
-   * @property {boolean} permission Whether the role or user has the permission to use this command
-   */
-
-  /**
-   * Fetches the permissions for this command.
-   * <warn>You must specify guildID if this command is handled by a {@link ApplicationCommandManager},
-   * including commands fetched for arbitrary guilds from it, otherwise it is ignored.</warn>
-   * @param {Snowflake} [guildID] ID for the guild to fetch permissions for if this is a global command
-   * @returns {Promise<ApplicationCommandPermissions[]>}
-   * @example
-   * // Fetch permissions for this command
-   * command.fetchPermissions()
-   *   .then(perms => console.log(`Fetched permissions for ${perms.length} users`))
-   *   .catch(console.error);
-   */
-  fetchPermissions(guildID) {
-    return this.manager.fetchPermissions(this, guildID);
-  }
-
-  /**
-   * Sets the permissions for this command.
-   * <warn>You must specify guildID if this command is handled by a {@link ApplicationCommandManager},
-   * including commands fetched for arbitrary guilds from it, otherwise it is ignored.</warn>
-   * @param {ApplicationCommandPermissionData[]} permissions The new permissions for the command
-   * @param {Snowflake} [guildID] ID for the guild to fetch permissions for if this is a global command
-   * @returns {Promise<ApplicationCommandPermissions[]>}
-   * @example
-   * // Set the permissions for this command
-   * command.setPermissions([
-   *   {
-   *     id: '876543210987654321',
-   *     type: 'USER',
-   *     permission: false,
-   *   },
-   * ])
-   *   .then(console.log)
-   *   .catch(console.error);
-   */
-  setPermissions(permissions, guildID) {
-    return this.manager.setPermissions(this, permissions, guildID);
+    return this.manager.delete(this, this.guildID ?? undefined);
```

Comment:
Why is this defaulting to undefined?

Reply:
Wow, that diff is messed up 😆 
Anyways, it was because I thought I had made a specific check for undefined, but I didn't, passing null is fine.